docs/applications/turn_channels_api.rst

```diff
@@ -9,16 +9,196 @@ Currently this only supports text message types. A future iteration will support
 Configuration
 ^^^^^^^^^^^^^
 
+The application has the following configuration options:
+
+.. py:data:: connector_name
+   :type: str
+
+   The name of the AMQP queue to publish outbound message on, and to consume inbound messages and events from. Required.
+
+.. py:data:: http_bind
+   :type: str
+
+   The address to bind the HTTP server to. Required.
+
+.. py:data:: vumi_base_url_path
+   :type: str
+
+   The base URL path for outbound message HTTP requests. Outbound message requests must be POSTed to ``<base_url_path>/messages``. Defaults to an empty string.
+
+.. py:data:: turn_base_url_path
```

Comment:
I'm wondering if we should change the name of this config. Since this would be something like `https://whatsapp.turn.io/v1/`, and `vumi_base_url_path` would be something like `/api/v1/`, which are very different types of configuration. We also probably shouldn't default to an empty string, since that will never be a valid config.